src/components/IconLibrary.js

```diff
@@ -2,10 +2,22 @@ import { library } from '@fortawesome/fontawesome-svg-core';
 import {
   fas,
   faStar as faStar1,
-  faStarHalfAlt
+  faStarHalfAlt,
+  faBus,
+  faHotel,
+  faCalendarWeek
```

Comment:
Is it cool if I change / play around with these later? I really like this library though, will be super useful!

Reply:
Totally!